cardano-cli/src/Cardano/CLI/Json/Friendly.hs

```diff
@@ -5,6 +5,9 @@
 {-# LANGUAGE ScopedTypeVariables #-}
 {-# LANGUAGE TypeApplications #-}
 {-# LANGUAGE TypeOperators #-}
+{-# OPTIONS_GHC -Wno-unrecognised-pragmas #-}
```

Comment:
Why is that required?

Reply:
https://github.com/IntersectMBO/cardano-cli/pull/856#discussion_r1716522158